src/filecoin/Filfox.ts

```diff
@@ -96,6 +96,31 @@ export const asFilfoxTransfer = asObject({
   value: asString
 })
 
+//
+// Account
+//
+
+export type FilfoxAccountResult = ReturnType<typeof asFilfoxAccountResult>
+export const asFilfoxAccountResult = asObject({
```

Comment:
This cleaner is too strict for new unused wallets where the response object is:

{"address":"f1pzdgw5mdxqnu6aixuy4bt4ygj6lhaajb3x7tgwi"}

New wallets will get stuck syncing